kpatch-build/kpatch-build

```diff
@@ -454,6 +454,7 @@ else
 		# upstream SUBLEVEL; they are always at SUBLEVEL 0
 		sed -i "s/^SUBLEVEL.*/${sublevel}/" "$SRCDIR/Makefile" || die
 		echo $ARCHVERSION > "$VERSIONFILE" || die
+		UBUNTU_KERNEL=1
```

Comment:
I think this will break debian.  Maybe this variable can be set in the `if [[ $DISTRO = ubuntu ]]` clause at line 415.